package.json

```diff
@@ -54,7 +54,7 @@
   },
   "devDependencies": {
     "@artsy/auto-config": "1.0.0",
-    "@artsy/palette": "4.14.38",
+    "@artsy/palette": "^4.14.38",
```

Comment:
I believe we want to pin the version since we have Renovate?

Reply:
hmmm- good catch. I would never have noticed that. While working on the palette stuff I believe I did `yarn upgrade @artsy/palette --latest` which must have done this.